tests/Block/AdminSearchBlockServiceTest.php

```diff
@@ -42,6 +42,11 @@ class AdminSearchBlockServiceTest extends BlockServiceTestCase
      */
     private $templateRegistry;
 
+    /**
+     * @var string
+     */
+    private $emptyBoxesOption;
```

Comment:
Why using a class property ? You could pass directly the value in the tests

Reply:
because I don't really know what I'm doing.. I copied the setup of other options but can simplify the test class if that's needed.